weekendhomework11/src/ba/bitcamp/sql/task1/Complaints.java

```diff
@@ -0,0 +1,68 @@
+package ba.bitcamp.sql.task1;
+
+/**
+ * This class present the complaint with specified attributes, used for get
+ * information of complaints from data base
+ * 
+ * @author Niddal.Salkic
+ *
+ */
+public class Complaints {
```

Comment:
Klasu bih nazvala Complaint, to je neki standard, a i jedan objekat opisuje jednu zalbu, tako da je ok.